### ISSUE TITLE
Having accidental duplicate FulltextSearch::enable() creates an error

### ISSUE THREAD

**Issue by Batsis12** [SilverStripers/elemental-seach#27]
If you have a duplicate `FulltextSearch::enable()` in your site or any of your module, it generates the below error on dev/build:

`Uncaught SilverStripe\ORM\Connect\DatabaseException: Couldn't run query: ALTER TABLE "File" DROP INDEX "SearchFields", ADD fulltext "SearchFields" ("Name","Title","Name","Title") 42S21-1060: Duplicate column name 'Name'
`

**Comment by fonsekaean** [comment]
Yes @Batsis12 this is a known problem. I never did anything about this as I thought its straightforward for the module to call `FulltextSearch::enable()`. But it'd be good to check whether any other module or the app had called it before the module doing it. Would you want to send a patch for that?
